hardhat.config.ts

```diff
@@ -52,7 +52,14 @@ export default {
     sources: "contracts",
   },
   solidity: {
-    version: "0.6.12",
+    compilers: [
+      {
+        version: "0.5.5",
+      },
```

Comment:
I'd add a comment to say that this is for compiling the WETH contract:
```suggestion
      {
        // used to compile WETH9.sol
        version: "0.5.5",
      },
```